src/components/mgt-person-card/mgt-person-card.ts

```diff
@@ -110,51 +110,286 @@ export class MgtPersonCard extends MgtTemplatedComponent {
-          <div class="default-view">
-            ${this.renderTemplate('default', { person: this.personDetails, personImage: image }) ||
-              html`
-                <mgt-person
-                  class="person-image"
-                  .personDetails=${this.personDetails}
-                  .personImage=${image}
-                ></mgt-person>
-                <div class="details">
-                  <div class="display-name">${user.displayName}</div>
-                  ${jobTitle} ${department}
-                  <div class="base-icons">
-                    ${this.renderIcons()}
-                  </div>
-                </div>
-              `}
-          </div>
-          <div class="additional-details-container">
-            ${this.renderAdditionalDetails()}
+    // Check for a person-details template
+    let personDetailsTemplate = this.renderTemplate('person-details', {
+      person: this.personDetails,
+      personImage: image
+    });
+    if (!personDetailsTemplate) {
+      const personImageTemplate = this.renderPersonImage(image);
+      const personNameTemplate = this.renderPersonName(person);
+      const contactIconsTemplate = this.renderContactIcons(person);
+
+      personDetailsTemplate = html`
+        <div class="image">
+          ${personImageTemplate}
+        </div>
+        <div class="details">
+          ${personNameTemplate}
+          <div class="base-icons">
+            ${contactIconsTemplate}
           </div>
         </div>
       `;
     }
+
+    const additionalDetailsTemplate = this.isExpanded
+      ? this.renderAdditionalDetails()
+      : this.renderAdditionalDetailsButton();
+
+    return html`
+      <div class="root" @click=${(e: Event) => this.handleClick(e)}>
+        <div class="person-details-container">
+          ${personDetailsTemplate}
+        </div>
+        <div class="additional-details-container">
+          ${additionalDetailsTemplate}
+        </div>
+      </div>
+    `;
+  }
+
+  /**
+   * Render the state when no data is available.
+   *
+   * @protected
+   * @returns {TemplateResult}
+   * @memberof MgtPersonCard
+   */
+  protected renderNoData(): TemplateResult {
+    return this.renderTemplate('no-data', null) || html``;
+  }
+
+  /**
+   * Render a display image for the person.
+   *
+   * @protected
+   * @param {*} image
+   * @memberof MgtPersonCard
+   */
+  protected renderPersonImage(imageSrc?: string): TemplateResult {
+    imageSrc = imageSrc || this.getImage();
+    return html`
+      <mgt-person class="person-image" .personDetails=${this.personDetails} .personImage=${imageSrc}></mgt-person>
+    `;
+  }
+
+  /**
+   * Render the display name and persona details (e.g. department, job title) for a person.
+   *
+   * @protected
+   * @param {IDynamicPerson} [person]
+   * @returns {TemplateResult}
+   * @memberof MgtPersonCard
+   */
+  protected renderPersonName(person?: IDynamicPerson): TemplateResult {
```

Comment:
Should we split this up even further?

* renderPersonName
* renderPersonTitle
* renderPersonSubtitle

Reply:
I agree. Fixed!